### ISSUE TITLE
Update and improve home page and tutorials

### ISSUE THREAD

**Issue by nguyentvan7** [frzyc/genshin-optimizer#316]
Once the UI changes are more stable, we can update the screenshots on the home page and rewrite some of the text to fix grammar errors/reflect new site features/changes.

We probably also want to redo the popup tutorials for each page to be either more concise, or not a popup at all. Maybe something like little tooltips that appear on the various UI elements, assuming that isn't too hard to do. This would be less mental overload for new users, and more interactive as well.

**Comment by frzyc** [comment]
I think the homepage has always been a tad half-assed. I believe as a homepage, its a tad too... wordy. There are also not a lot of distinctions between the sections. Quite a lot of questions needs to be answered:
- should it be an index? it only links to like 4 out of our 6 pages, there is a dup link as well, a lot of sub pages(specific character pages) shouldn't be showcased? it is the only accessible link to the scanners page...
- separate about page? 
- Intro is too wordy(how and where to we explain the caveats?)
- the discord widget serves no purpose, just a big ugly button to discord server.
- layout is uninspired and chunky


My opinion is that we should define the requirements of a "welcome/index" page, and try to condense the content according to those. I think a lot of the bloat here should be moved to another page (about? tooltips?), and design a clean, concise home page.

**Comment by frzyc** [comment]
I think the home page should be more of a dashboard page. 
- resin counter(link to tools)
- server time( use timezone selected in tools)
- farmables today(need to hook up ascension mats from character sheet from datamine, but it would only show char/weapons added to GO, that arnt fully ascended.)
- overview of # of arts, characters, weapons in inventory
  - show characters that doesn't have build as quick click to optimize?
- excerpt of patch notes(latest minor?)
- current banner?
- latest guide videos(instead of just link to playlist, embed videos?)

**Comment by frzyc** [comment]
Resolved by `8.10.0`
